api/src/functions/express/app.ts

```diff
@@ -168,11 +170,24 @@ const dynamicsElevatorSafetyInspectionClient = DynamicsElevatorSafetyInspectionC
   logger,
   DYNAMICS_ELEVATOR_SAFETY_URL
 );
+const dynamicsElevatorSafetyRegistrationClient = DynamicsElevatorSafetyRegistrationClient(
+  logger,
+  DYNAMICS_ELEVATOR_SAFETY_URL
+);
 
-const dynamicsElevatorSafetyClient = DynamicsElevatorSafetyClient(logger, {
+const dynamicsElevatorSafetyInspectionStatusClient = DynamicsElevatorSafetyInspectionStatusClient(logger, {
   accessTokenClient: dynamicsElevatorSafetyAccessTokenClient,
   elevatorSafetyInspectionClient: dynamicsElevatorSafetyInspectionClient,
 });
+const dynamicsElevatorSafetyRegistrationStatusClient = DynamicsElevatorSafetyRegistrationStatusClient(
+  logger,
+  {
+    accessTokenClient: dynamicsElevatorSafetyAccessTokenClient,
```

Comment:
[dust]
```suggestion
    elevatorAccessTokenClient: dynamicsElevatorSafetyAccessTokenClient,
```

Reply:
this would break the established form for dynamics access token- the only reason that housing is called out in this client is because of the additional dependancy